# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../icc/pkg_make_globh.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./dir.c.orig	Fri Jun 21 18:44:17 2002
+++ ./dir.c	Fri Jun 21 18:47:57 2002
@@ -994,6 +994,7 @@
 
 /* Hooks for globbing.  */
 
+#define _FILE_OFFSET_BITS 32
 #include <glob.h>
 
 /* Structure describing state of iterating through a directory hash table.  */
--- ./read.c.orig	Fri Jun 21 18:48:34 2002
+++ ./read.c	Fri Jun 21 18:48:45 2002
@@ -21,6 +21,7 @@
 
 #include <assert.h>
 
+#define _FILE_OFFSET_BITS 32
 #include <glob.h>
 
 #include "dep.h"
